Languages/IronPython/IronPython/Runtime/Operations/PythonTypeOps.cs

```diff
@@ -720,9 +721,9 @@ internal struct BuiltinFunctionKey {
 
         internal static string GetDocumentation(Type type) {
             // Python documentation
-            object[] docAttr = type.GetCustomAttributes(typeof(DocumentationAttribute), false);
-            if (docAttr != null && docAttr.Length > 0) {
-                return ((DocumentationAttribute)docAttr[0]).Documentation;
+            var docAttr = type.GetTypeInfo().GetCustomAttributes(typeof(DocumentationAttribute), false);
```

Comment:
Yep. Replaced Length and indexing with Linq since on .NET Core the return type is an IEnumerable.